website/src/contribute/release-guide.md

```diff
@@ -691,6 +691,15 @@ done
 ./gradlew :sdks:go:container:dockerPush -Pdocker-tag=${RELEASE}_rc{RC_NUM}
 ```
 
+* Build Flink job server images and push to DockerHub.
+
+```
+FLINK_VER=("1.7", "1.8", "1.9")
```

Comment:
Could you point to a source-of-truth for the list of supported version?

Reply:
Good question... I was just basing this off the info on the website. I don't see a legit source of truth anywhere in the code. https://beam.apache.org/documentation/runners/flink/#version-compatibility

@mxm might know more.

Reply:
Something like the following could do:

```bash
FLINK_VER=($(ls -1 runners/flink | awk '/^[0-9]+\.[0-9]+$/{print}'))
```

Reply:
Perhaps a sanity check that the array length is greater 0 would also be useful.

Reply:
Since all the Flink Runner versions are independent projects at the moment (sharing a base template), I'm not sure there is a better solution.

Reply:
My workspace contains directories runners/flink/1.5 and 1.6. I think we might be better off hard-coding it.

Reply:
I would incline towards hardcoding and if possible have a separate validation in the release guide or a test to make sure we produce output for the right versions.

Reply:
>My workspace contains directories runners/flink/1.5 and 1.6. I think we might be better off hard-coding it.

That would be incorrect in case of a release. You need to clone the directory to receive a clean checkout.

>I would incline towards hardcoding and if possible have a separate validation in the release guide or a test to make sure we produce output for the right versions.

This will produce the right output. If you have stale directories it will fail which is correct.

There are pro and cos to both variants, but I'd say the risk is higher that we forget to update this list, than the script returning something incorrect, which should always lead to a failure.

Reply:
Just a suggestion, feel free to go ahead with what you think is best.

Reply:
> I'd say the risk is higher that we forget to update this list

That's a good point. And in the worst case scenario, even if there are stale directories, it would be presumably easier to delete the unsupported containers than to build new ones.

Reply:
Included your suggestions @mxm